pootle/templates/staticpages/_body.html

```diff
@@ -9,7 +9,14 @@
   {% endblocktrans %}
   </div>
   {% endif %}
-  <h1 class="title">{{ page.title }}</h1>
+  <h1 class="title">
+    {{ page.title }}
+    {% if user.is_superuser %}
```

Comment:
The following allows to check if a specific user can administrate a TP:

```python
check_user_permission(user, 'administrate', tp.directory)
```

I cannot find any specific permissions for administrating a project or a language, so perhaps you should use the `change_project` and `change_language` permissions.